packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendar.m

```diff
@@ -127,5 +129,27 @@ - (void)setRct_selectedDates:(NSArray<NSDate *> *)rct_selectedDates {
     [super setSelectedDates:simpleDates];
 }
 
+- (void)setRct_colorBuckets:(NSArray<RCTBPKColorBucket *> *_Nullable)rct_colorBuckets {
+    if(_rct_colorBuckets == rct_colorBuckets) {
+        return;
+    }
+
+    _rct_colorBuckets = rct_colorBuckets;
+
+    for (int i = 0; i < rct_colorBuckets.count ; i+= 1) {
+        RCTBPKColorBucket *bucket = rct_colorBuckets[i];
+
+        NSMutableArray *localDates = [[NSMutableArray alloc]initWithCapacity:bucket.days.dates.count];
+        for (int j = 0; j < bucket.days.dates.count ; j+= 1) {
+            localDates[j] = [RCTBPKCalendarDateUtils convertDateToLocal:bucket.days.dates[j]
```

Comment:
Can we cleanup the method call syntax here, maybe align the colons or whatever Xcode does by default?